client/src/handlebars/register_helpers.side-effects.js

```diff
@@ -211,52 +211,37 @@ Handlebars.registerHelper("will_change_to", function (val, formatter, context) {
 Handlebars.registerHelper("will_change_by", function (val, formatter, context) {
   return en_master_change(val, formatter, "by", "future");
 });
-Handlebars.registerHelper("fr_changed_to", function (
-  val,
-  genre,
-  nombre,
-  formatter,
-  context
-) {
-  return fr_master_change(val, formatter, "to", "past", genre, nombre);
-});
-Handlebars.registerHelper("fr_changed_by", function (
-  val,
-  genre,
-  nombre,
-  formatter,
-  context
-) {
-  return fr_master_change(val, formatter, "by", "past", genre, nombre);
-});
+Handlebars.registerHelper(
+  "fr_changed_to",
+  function (val, genre, nombre, formatter, context) {
+    return fr_master_change(val, formatter, "to", "past", genre, nombre);
+  }
+);
+Handlebars.registerHelper(
+  "fr_changed_by",
+  function (val, genre, nombre, formatter, context) {
+    return fr_master_change(val, formatter, "by", "past", genre, nombre);
+  }
+);
```

Comment:
Did prettier decide to reformat this file for some reason? Do you know why?

Reply:
Yeah at some point I had encountered a similar error that Christina (I think) was having with prettier and husky or something. The file needed to be reformatted and committed so that I could push, no clue why it popped up when it did.